docs/AboutUs.md

```diff
@@ -14,18 +14,17 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/wkxcass.png" width="200px">
 
 [[github](https://github.com/wkxcass)]
-[[portfolio](team/kexin.md)]
+[[portfolio](team/wkxcass)]
```

Comment:
Maybe can add .md after the github username, otherwise the button wouldn't link to the md files if im not wrong :)

Reply:
Ah ok nice catch! My refactoring tool missed this.. Thanks!!